zvariant/src/type.rs

```diff
@@ -77,6 +87,121 @@ pub trait DynamicDeserialize<'de>: DynamicType {
         S::Error: Into<zvariant::Error>;
 }
 
+/// Implements the [`Type`] trait by delegating the signature
+/// to a simpler type (usually a tuple).
+/// Tests that ensure that the two types are serialize-compatible
+/// are auto-generated.
```

Comment:
Let's only break lines (including comments) at 100 chars limit.

```suggestion
/// Implements the [`Type`] trait by delegating the signature to a simpler type (usually a tuple).
/// Tests that ensure that the two types are serialize-compatible are auto-generated.
```